example/screens/PreviewScreen/index.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  CaptureDevice,
+  TrackEncoding,
+  useCamera,
+  useMicrophone,
+} from '@fishjam-dev/react-native-client';
+import BottomSheet from '@gorhom/bottom-sheet';
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React, { useEffect, useRef, useState } from 'react';
+import {
+  Button,
+  Platform,
+  SafeAreaView,
+  StyleSheet,
+  TouchableWithoutFeedback,
+  View,
+} from 'react-native';
+
+import { SwitchCameraButton } from './SwitchCameraButton';
+import { SwitchOutputDeviceButton } from './SwitchOutputDeviceButton';
+import { ToggleCameraButton } from './ToggleCameraButton';
+import { InCallButton } from '../../components';
+import LetterButton from '../../components/LetterButton';
+import { NoCameraView } from '../../components/NoCameraView';
+import { SoundOutputDevicesBottomSheet } from '../../components/SoundOutputDevicesBottomSheet';
+import VideoPreview from '../../components/VideoPreview';
+import { usePreventBackButton } from '../../hooks/usePreventBackButton';
```

Comment:
Maybe lets add path so import would look like:
`import { usePreventBackButton } from '@hooks/usePreventBackButton';`

Reply:
50/50 for adding additional project config to have 'nicer' imports. But we can improve directory structure to have separate folder with JS source (`example/src/...`)